wagtailio/static/sass/base/_typography.scss

```diff
@@ -76,6 +76,27 @@ h4,
     }
 }
 
+.teaser-heading {
+    font-size: 1.25rem;
+    line-height: 1.5rem;
+    font-weight: $weight--bold;
+
+    @include media-query(large) {
+        font-size: 1.875rem;
+        line-height: 2.438rem;
+    }
+}
+
+.teaser-intro {
+    font-size: 1rem;
+    line-height: 1.5rem;
+
+    @include media-query(large) {
+        font-size: 1.25rem;
+        line-height: 2.5rem;
```

Comment:
In the Figma file this is shown at font-size: 20px / line-height: 28px:

![Screenshot 2022-07-27 at 14 03 23](https://user-images.githubusercontent.com/31627284/181255498-622eaadf-1829-488c-91fb-7351779fbe49.png)

Reply:
Thanks for the spot.